art/config.json

```diff
@@ -5,7 +5,7 @@
                 "experiment.js",
                 "style.css",
                 "static/js/jspsych/plugins/jspsych-call-function.js",
-                "static/js/jspsych/custom_plugins/jspsych-multi-button.js"
+                "static/js/jspsych/custom_plugins/jspsych-single-stim-button.js"
```

Comment:
I don't see this script in battery folder:

https://github.com/expfactory/expfactory-battery/tree/master/static/js/jspsych/custom_plugins